### DISCUSSION TITLE
Login Logout

### DISCUSSION THREAD

**Discussion by CollRen** [582-41E-MA/projet-web-2-CollRen#38] [Show and tell]
@claraquintela C'est long faire le login logout. Je pense terminer jeudi avant-midi tout au plus.

**Comment by claraquintela** [comment]
Pas de trouble.

Je suis encore bien ocupée dans lle privilege.
J’ai galeré aujourd’hui, mais je crois avoir finir le “create new privilège
“. Demain je teste la traduction :)
